projects/packages/jetpack-mu-wpcom/src/features/wpcom-admin-interface/wpcom-admin-interface.php

```diff
@@ -409,6 +409,11 @@ function wpcom_is_duplicate_views_experiment_enabled() {
 
 	$variation = get_user_option( RDV_EXPERIMENT_FORCE_ASSIGN_OPTION, get_current_user_id() );
 
+	/**
+	 * We cache it for both AT and Simple because we want to give a12s to be able to switch between variations for their accounts - this can be useful during support.
```

Comment:
Can we use ExPlat to switch between variations? Or will we be forced to use the escape hatch (p7DVsv-m73-p2)?

Reply:
Never mind. Just realized that we cannot rely on ExPlat because we're enforcing `treatment`, so the escape hatch is the only alternative.